tests/system_tests_http2_tls.py

```diff
@@ -593,3 +593,101 @@ def test_listener_mgmt_missing_ssl_profile(self):
 
     def test_listener_mgmt_missing_ca_file(self):
         self._test_listener_mgmt_missing_ca_file()
+
+
+class Http2TestTlsOverTcpTwoRouter(Http2TestBase, CommonHttp2Tests, RouterTestSslBase):
+    """
+    In this two router test, the tcpListener is on Router QDR.A and the tcpConnector is on router
+    QDR.B. Both the listener and the connector are encrypted. The server that QDR.B connects to
+    is also encrypted.
+    Client authentication is required for curl to talk to the router QDR.A listener http port.
```

Comment:
I don't see how the two test methods below in this class perform what's outlined in this comment.

Ahh, I actually do see it now, the actual tests are inherited into this, and the two test methods below are actually test teardown! Right.

Reply:
/me feels like I already made a comment like this in the past; advice such like https://testing.googleblog.com/2019/12/testing-on-toilet-tests-too-dry-make.html sounds great on paper, but I never saw this work in practice, in a test suite, over time... ;(